src/ui/components/shared/WorkspaceSettingsModal/OrganizationSettings.tsx

```diff
@@ -0,0 +1,172 @@
+  const ref = useRef<NodeJS.Timeout | undefined>();
+
+  useEffect(() => {
+    if (ref.current) {
+      clearTimeout(ref.current);
+    }
+
+    ref.current = setTimeout(() => {
+      updateWorkspaceSettings({
+        variables: {
+          workspaceId,
+          motd: message,
+        },
+      });
+    }, 500);
+  }, [message]);
+
+  const updateFeature = (features: Partial<WorkspaceSettings["features"]>) => {
+    updateWorkspaceSettings({
+      variables: {
+        workspaceId,
+        features,
+      },
+    });
+  };
+
+  if (!workspace) {
+    return null;
+  }
+
+  const disabled = !workspace.isOrganization;
+
+  return (
+    <div className={classNames("space-y-4", { "text-gray-500": disabled })}>
+      {disabled ? (
+        <p>
+          These features are only available to teams on our Organization Plan. Want to upgrade? Get
+          in touch!
+        </p>
+      ) : null}
+      <div className="text-xs uppercase font-semibold">Recordings</div>
+      <Row>
+        <Label>Disable Public Recordings</Label>
+        <Input>
+          <label className="flex items-center" htmlFor="disable_public_recordings">
+            <input
+              className={classNames("rounded-sm ml-0 text-sm", {
+                "bg-toolbarBackground": disabled,
+                "border-gray-300": disabled,
+              })}
+              disabled={disabled}
+              type="checkbox"
+              id="disable_public_recordings"
+              name="disable_public_recordings"
+              onChange={e => updateFeature({ recording: { public: !e.currentTarget.checked } })}
+              checked={!workspace.settings.features.recording.public}
+            />
+          </label>
+        </Input>
+      </Row>
+      {/* <Row>
+        <Label>Allow Recordings From</Label>
+        <Input>
+          <input
+            className={classNames("rounded-md mr-2 w-full text-sm", {
+              "bg-toolbarBackground": disabled,
+              "border-gray-300": disabled,
+            })}
+            disabled={disabled}
+            placeholder={`staging.${workspace?.domain}`}
+            type="text"
+          />
+        </Input>
+      </Row>
+      <Row>
+        <Label>Block Recordings From</Label>
+        <Input>
+          <input
+            disabled={disabled}
+            placeholder={`production.${workspace?.domain}`}
+            type="text"
+            className={classNames("rounded-md w-full mr-2 text-sm", {
+              "bg-toolbarBackground": disabled,
+              "border-gray-300": disabled,
+            })}
+          />
+        </Input>
+      </Row> */}
+      <div className="text-xs uppercase font-semibold">Members</div>
+      <Row>
+        <Label>Disable My Library</Label>
+        <Input>
+          <label className="flex items-center" htmlFor="restrict_users_to_domain">
+            <input
+              className={classNames("rounded-sm ml-0 text-sm", {
+                "bg-toolbarBackground": disabled,
```

Comment:
Collapse the two lines here too.

Reply:
This is a bit of style pref, imo. I actually like it this was as it ties single classes to the variable but not passionate about it. :)